server/src/webserver.rs

```diff
@@ -645,18 +656,30 @@ async fn serve_file(
         source_lexer(&file_contents, lexer)
     };
 
-    // <p>Convert doc blocks from Markdown to HTML</p>
+	// <p>Convert doc blocks from Markdown to HTML then parse using HTML5Ever</p>
     let mut options = Options::empty();
     options.insert(Options::ENABLE_STRIKETHROUGH);
     for code_doc_block in &mut code_doc_block_arr {
         if let CodeDocBlock::DocBlock(ref mut doc_block) = code_doc_block {
             let parser = Parser::new_ext(&doc_block.contents, options);
             let mut html_output = String::new();
             html::push_html(&mut html_output, parser);
-            doc_block.contents = html_output;
+		
+		// <p>Create a new RcDom object to represent the DOM tree with default settings</p>
```

Comment:
Here and following, switch tabs to spaces.

Reply:
This was fixed with cargo fmt.  Please see updated code.